src/pages/dho/Plan.vue

```diff
@@ -190,6 +203,7 @@ export default {
 
 <template lang="pug">
 .page-plan(v-if="!loading")
+  DowngradePopUp(:value="isDowngradePopUpOpen" @activatePlan="activatePlan" @hidePopUp="state = 'BILLING'")
```

Comment:
@evgenibir just small change if you can lowercase the component name as we always used that convetion :)